src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java

```diff
@@ -483,6 +543,14 @@ private ComponentService.TranslationsDTO getMessages(final Locale locale, final
             return new ComponentService(dto).getTranslations(null);
     }
 
+    private ComponentService.TranslationsDTO getMultiVersionMessagesOfKey(final Locale locale, final String version, final String component, final String key, boolean useLocaleFallback) {
```

Comment:
this function can be combined with getMultiVersionMessageWithArgs as the logic is not common.